server.js

```diff
@@ -0,0 +1,15 @@
+const express = require('express');
+const app = express();
```

Comment:
It would be good to add error handling middleware to handle any uncaught exceptions or errors. This can be done by adding a middleware function at the end of the middleware function stack. [important]